### PR TITLE
Remove juju install section and other copy updates

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Updated according to [copy docs](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit)

## QA

- Check https://ubuntu-com-13179.demos.haus/landscape/install against [copy](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit)
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6422

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/landscape/install.html [hunk 1]
```diff
@@ -379,26 +178,23 @@ <h3 class="p-heading--2">Install</h3>
...
                   <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>sudo add-apt-repository ppa:landscape/self-hosted-23.03 </code></pre>
+                    <pre class="p-code-snippet__block is-wrapped"><code>sudo add-apt-repository -y ppa:landscape/self-hosted-23.03 </code></pre>
                   </div>
                 </li>
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Install Landscape</p>
                   <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>sudo apt update && sudo apt install landscape-server-quickstart</code></pre>
+                    <pre class="p-code-snippet__block is-wrapped"><code>sudo apt update sudo DEBIAN_FRONTEND=noninteractive apt-get install -y landscape-server-quickstart</code></pre>
```
Comment (rajannpatel) [inline templates/landscape/install.html:197]: I made a mistake in the copydoc, which I just corrected, this line should read: `sudo apt update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y landscape-server-quickstart` It is missing the `&&`

File: templates/landscape/install.html [hunk 2]
```diff
@@ -424,10 +220,16 @@ <h3 class="p-heading--2">Configure SSL</h3>
                     <pre class="p-code-snippet__block is-wrapped"><code>sudo snap install certbot --classic</code></pre>
                   </div>
                 </li>
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Set your email address in a variables</p>
```
Comment (rajannpatel) [inline templates/landscape/install.html:224]: Please revise this to use a singular form of the word "variable": `Set your email address in a variable` (remove trailing `s`)

File: templates/landscape/install.html [hunk 3]
```diff
@@ -448,9 +250,10 @@ <h3 class="p-heading--2">Configure postfix for email</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Set SMTP information in variables</p>
+                  <p>Use an API Key from: https://app.sendgrid.com/settings/api_keys</p>
                   <div class="p-code-snippet">
                     <pre class="p-code-snippet__block is-wrapped"><code>SMTP_HOST='smtp.sendgrid.net'
-SMTP_PORT='465'
+SMTP_PORT='587'
```
Comment (rajannpatel) [inline templates/landscape/install.html:256]: below, both of the following list items "Download an SSL CA file" and "Set appropriate permissions" need to be removed

File: templates/landscape/install.html [hunk 4]
```diff
@@ -622,7 +429,7 @@ <h3 class="p-heading--2">Configure</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Define parameters in variables</p>
-                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their account name.</p>
+                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their <u>account name</u> set LANDSCAPE_FQDN as <code>landscape.canonical.com</code>.</p>
```
Comment (rajannpatel) [inline templates/landscape/install.html:432]: `<u>account name</u>` should really be a hyperlink to: https://landscape.canonical.com/settings

File: templates/landscape/install.html [hunk 5]
```diff
@@ -685,7 +492,7 @@ <h3 class="p-heading--2">Configure</h3>
...
```
Comment (rajannpatel) [inline templates/landscape/install.html:495]: `<u>account name</u>` should really be a hyperlink to: https://landscape.canonical.com/settings

File: templates/landscape/install.html [hunk 6]
```diff
@@ -715,6 +522,14 @@ <h3 class="p-heading--2">Install</h3>
             </div>
             <div class="p-strip u-no-padding--top">
               <ol class="p-list--divided">
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Define parameters in variables</p>
+                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their <u>account name</u> set LANDSCAPE_FQDN as <code>landscape.canonical.com</code>.
```
Comment (rajannpatel) [inline templates/landscape/install.html:527]: `<u>account name</u>` should really be a hyperlink to: https://landscape.canonical.com/settings

File: templates/landscape/install.html [hunk 7]
```diff
@@ -752,29 +567,28 @@ <h3 class="p-heading--2">Configure</h3>
...
             </div>
           </div>
           <hr class="u-no-margin--bottom u-hide--small">
           <div class="col-3 col-medium-1 u-hide--small">
             <p class="p-step-counter"></p>
           </div>
           <div class="col-9 col-medium-5">
-            <p class="p-heading--5">Install within the runcmd section of Cloud Init</p>
+            <p class="p-heading--5">Configure landscape-client</p>
+            <p>To discover additional supported client keys visit the <a href="https://cloudinit.readthedocs.io/en/latest/reference/modules.html#landscape">cloud-init Landscape module documentation</a>.</p>
```
Comment (rajannpatel) [inline templates/landscape/install.html:585]: I had omitted this in the copydoc originally, and I added it in when I noticed it was missing:
`Self-hosted Landscape users should set account_name as `standalone` and define the url and ping_url keys. Landscape SaaS customers should specify their <a href="https://landscape.canonical.com/settings">account name</a>, and the url and ping_url keys can be omitted.`

File: templates/landscape/install.html [hunk 8]
```diff
@@ -182,183 +157,7 @@ <h2>Install Landscape Server to manage machines and containers</h2>
       </div>
       <hr class="is-fixed-width">
 
-      <div class="p-tabs__content" tabindex="0" role="tabpanel" id="automated-tab" aria-labelledby="automated" style="scroll-margin-top:86px;">
-        <div class="row">
-          <div class="col-8 col-start-large-4 p-strip is-shallow">
-            <p>The entire installation and configuration can be automated with <a href="https://cloud-init.io/">Cloud Init</a>, using this <a href="https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init.yaml">cloud-init.yaml</a> file.</p>
```
Comment (rajannpatel) [inline templates/landscape/install.html:188]: I am told "Cloud Init" should be written as "Cloud-init" here. I have made this correction in the copydoc as well.

File: templates/landscape/install.html [hunk 9]
```diff
@@ -448,9 +250,10 @@ <h3 class="p-heading--2">Configure postfix for email</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Set SMTP information in variables</p>
+                  <p>Use an API Key from: https://app.sendgrid.com/settings/api_keys</p>
```
Comment (rajannpatel) [inline templates/landscape/install.html:253]: I rephrased "Use an API key from" to instead read as:
```
The value of SMTP_PASSWORD should be the SendGrid API Key from
```

File: templates/landscape/install.html [hunk 10]
```diff
@@ -182,183 +157,7 @@ <h2>Install Landscape Server to manage machines and containers</h2>
...
-                  </div>
-                </li>
-              </ol>
-            </div>
-          </div>
-        </div>
-      </div>
-
-      <div class="p-tabs__content u-hide" tabindex="0" role="tabpanel" id="quickstart-tab" aria-labelledby="quickstart" style="scroll-margin-top:86px;">
+      <div class="p-tabs__content" tabindex="0" role="tabpanel" id="quickstart-tab" aria-labelledby="quickstart" style="scroll-margin-top:86px;">
```
Comment (rajannpatel) [inline templates/landscape/install.html:160]: Below we have this line: "sudo apt update && sudo apt install -y wget software-properties-common"
The word "wget" should be removed from there, and it should be replaced with "ca-certificates", it should read:
```
sudo apt update && sudo apt install -y ca-certificates software-properties-common 
```

**Review by rajannpatel — CHANGES_REQUESTED** [review]
A few minor changes are requested, details are included in the comments.

File: templates/landscape/install.html (hunk 10 above)
Comment (carkod) [reply templates/landscape/install.html:160]: Can you add `ca-certificates` to the copy docs? I believe it's missing too?

**Comment by carkod** [comment]
> A few minor changes are requested, details are included in the comments.

Thanks. Addressed.

File: templates/landscape/install.html (hunk 10 above)
Comment (rajannpatel) [reply templates/landscape/install.html:160]: i checked for it, and do see it in the copydocs:
image

**Comment by rajannpatel** [comment]
I took another look on the preview page. I noticed `PRETTY_HOSTNAME="My Landscape Server"` should be removed, so I updated the copydoc accordingly.

At the top of the page there are 4 bullet points in the copydoc, but on the preview page I see the 4th one is missing. It reads `DNS administration access for the hostname you will use to access Landscape is necessary, if you wish to use LetsEncrypt to obtain an SSL certificate.`

After these 2 changes, I think we can consider this 100% complete, and ready to go live!

**Comment by carkod** [comment]
> DNS administration access for the hostname you will use to access Landscape is necessary, if you wish to use LetsEncrypt to obtain an SSL certificate

Done.
I think that was just missing the separation line.
Thanks.

**Review by rajannpatel — APPROVED** [review]

File: templates/landscape/install.html
Comment (pedoch) [inline templates/landscape/install.html:1]: - [ ] In the paragraph on line `590` is "Clout Init" is supposed to be "Cloud-init"

**Review by pedoch — APPROVED** [review]
Thank you @carkod! Looks really good. Just a couple of comments.

File: templates/landscape/install.html [hunk 11]
```diff
@@ -448,25 +250,14 @@ <h3 class="p-heading--2">Configure postfix for email</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Set SMTP information in variables</p>
+                  <p>The value of SMTP_PASSWORD should be the SendGrid API Key from: https://app.sendgrid.com/settings/api_keys</p>
```
[the remark this answers is not in the document]
Comment (rajannpatel) [reply templates/landscape/install.html:253]: I am OK with linking the words `SendGrid API Key` and removing `from: https://app.sendgrid.com/settings/api_keys`. It's a better user experience. We can proceed with this, and am tagging @gloriaquintanilla for awareness on this decision.

[the remark this answers is not in the document]
Comment (rajannpatel) [reply templates/landscape/install.html:253]: The copydoc has been updated to reflect this.

File: templates/landscape/install.html [hunk 12]
```diff
@@ -715,6 +510,14 @@ <h3 class="p-heading--2">Install</h3>
             </div>
             <div class="p-strip u-no-padding--top">
               <ol class="p-list--divided">
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Define parameters in variables</p>
+                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their <a href="https://landscape.canonical.com/settings">account name</a> and set LANDSCAPE_FQDN as <code>landscape.canonical.com</code>.
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/landscape/install.html:515]: Good catch

**Pull request merged**